Content/Application/MetadataResolver/MetadataResolver.php

```diff
@@ -46,7 +46,7 @@ public function resolveItems(array $items, array $data, string $locale): array
                     $this->resolveItems($item->getItems(), $data, $locale)
                 );
             } else {
-                $contentViews[$name] = $this->resolveProperty($type, $data[$name], $locale, ['metadata' => $item]);
+                $contentViews[$name] = $this->resolveProperty($type, $data[$name] ?? null, $locale, ['metadata' => $item]);
```

Comment:
we should may not forgot about moving the metadata out of params to only have scalar values in the params which we forward to view. but that unrelated to this PR.

Reply:
added it to the PR description https://github.com/sulu/sulu/pull/7611